### DISCUSSION TITLE
Logging for specific class/trigger

### DISCUSSION THREAD

**Discussion by dayakumbhar** [jongpie/NebulaLogger#486] [Q&A]
Is there a capability to enable logging for only specific classes/trigger and disable all other logs?  Meaning, from the org level setting, I want to keep it enabled. The log statement may be written across classes, however, I want to only log on a need basis for a given class or trigger when I configured so in the settings. 
I believe this capability is not there currently, however, would like to confirm.

**Comment by jongpie** [comment]
Hi @dayakumbhar - you're correct, this capability is not currently available. I briefly explored this idea 1-2 years ago, but I wasn't thrilled with the prototype that I created, so I never fully implement this feature. But, I think it could still be a useful feature to have- I'll look into this further later this year to see if there's a good way to implement it.
